python/grep.py

```diff
@@ -417,8 +420,7 @@ def get_config_log_filter():
 
 def get_home():
     home = weechat.config_string(weechat.config_get('logger.file.path'))
-    home = path.abspath(path.expanduser(home))
-    return home.replace('%h', weechat.info_get('weechat_dir', ''))
+    return weechat.string_eval_path_home(home, '', '', '')
```

Comment:
Why does this show the previous changeset? The fix asked by @m4v doesn't use `weechat.string_eval_path_home` anymore.